dev/cpp-dev/code.cpp

```diff
@@ -0,0 +1,63 @@
+#include <Rcpp.h>
+#include <vector>
+using namespace Rcpp;
+using namespace std;
+
+
+// from https://coolbutuseless.github.io/2018/09/17/intersection-of-multiple-vectors/
+// [[Rcpp::export]]
+std::vector<int> intersection_rcpp_core(IntegerVector v1, IntegerVector v2) {
```

Comment:
Functions in development for Rcpp to speed things up. For some reason slower when inside of a package. Needs further development.